dashboard/app/controllers/pd/professional_learning_landing_controller.rb

```diff
@@ -62,4 +35,37 @@ def applications_closed
     closed = Rails.env.production? && !current_user.try(:workshop_admin?) && Gatekeeper.disallows('pd_teacher_application')
     render json: closed
   end
+
+  # Returns non-ended workshops the user is facilitating.
+  def workshops_as_facilitator_for_pl_page
+    workshops_as_facilitator =
+      User.find(params[:user_id]).
```

Comment:
should we use the authenticity token to get the current_user for these endpoints rather than passing a param?

Reply:
That works for me! Do you know of an example of where we do this by chance? I'm not sure I know how to do that.

Reply:
I think all you need to do is add the 'X-CSRF-Token' header with the `getAuthenticityToken` helper function. example here: https://github.com/code-dot-org/code-dot-org/blob/staging-next/apps/src/schoolInfo/utils/updateSchoolInfo.ts#L29

and then you can see the controller method has access to the user through the `current_user` function that I think comes from devise: https://github.com/code-dot-org/code-dot-org/blob/staging-next/dashboard/app/controllers/api/v1/user_school_infos_controller.rb#L25

the reason for getting the user_id from the authenticated user object rather than passing it as a param is to prevent users accessing another user's workshops. not sure if that's a true security concern but it's good practice either way

Reply:
Oh I didn't realize that's what that token did! Thank you, I'll definitely fix that!